xchange-lgo/src/main/java/org/knowm/xchange/lgo/dto/trade/LgoUserTrade.java

```diff
@@ -0,0 +1,73 @@
+package org.knowm.xchange.lgo.dto.trade;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LgoUserTrade {
+
+  private final String id;
+  private final String orderId;
+  private final String productId;
+  private final String price;
+  private final String quantity;
```

Comment:
parsing to BigDecimals, Dates, etc. should be done during JSON deserialization, NOT in the adapters

Reply:
👍